### ISSUE TITLE
Adding tabs for recent files 

### ISSUE THREAD

**Issue by pankaj-2503** [processing/p5.js-web-editor#2885]
### Increasing Access

It would  make  easy to navigate through different files easily so that  user don't need to navigate via folder structure.

### Most appropriate sub-area of p5.js?

- [X] Accessibility
- [ ] Color
- [X] Core/Environment/Rendering
- [ ] Data
- [X] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### Feature request details

![image](https://github.com/processing/p5.js/assets/90036864/53b387fd-8b44-4e1c-8842-af3a52022d50)

**Comment by AdityyaX** [comment]
Hey, @pankaj-2503  could you provide more details about the specific issue? and steps to replicate it .

**Comment by Aseem22** [comment]
Is it something that still needs to be implemented? If so, I would be happy to work on it. @lindapaiste @raclim

**Comment by raclim** [comment]
@Aseem22 I'm not entirely sure yet if this is a strong need for this feature, and probably still requires more input from other folks. 

However, I think it could be helpful to provide more ideas on details for what this feature would look like to help come to that decision. (For example, what would the tabs look like? What are the maximum amount of tabs we'd allow? etc.) If you have any suggestions for a design or even a light prototype that folks could look at, I think that could be great place to get started!

**Comment by Aseem22** [comment]
@raclim I think this feature is required for improved navigation and efficiency especially when working with lot of files and when there's often need to reference other files.
One important feature can be when a user returns to the site, their recent tabs should be restored, maintaining their workflow continuity.
The tabs could be similar to recent tabs we see on VS Code(or browsers), with a consistent look and feel that matches our website's design. Each tab would display the file name and possibly an icon indicating the file type. We can allow as much tabs as user want (a carousel of recent tabs).
